libc/hdr/stdio_overlay.h

```diff
@@ -27,6 +27,17 @@
 #undef _FORTIFY_SOURCE
 #endif
 
+#ifdef __USE_EXTERN_INLINES
+#define LIBC_OLD_EXTERN_INLINES __USE_EXTERN_INLINES
```

Comment:
`#define LIBC_OLD_EXTERN_INLINES` is enough, system headers do not set any value for `__USE_EXTERN_INLINES`